spec/models/user_spec.rb

```diff
@@ -43,18 +43,26 @@
   end
 
   describe "#follows?"
-    it "returns true if the user has a following relationship with another user" do 
+    it "returns true if the user has a following relationship with another user" do
       alice = Fabricate(:user)
       bob = Fabricate(:user)
       Fabricate(:relationship, leader: bob, follower: alice)
       expect(alice.follows?(bob)).to be_truthy
     end
 
-    it "returns false if the user does not have a following relationship with another user" do 
+    it "returns false if the user does not have a following relationship with another user" do
       alice = Fabricate(:user)
       bob = Fabricate(:user)
       Fabricate(:relationship, leader: alice, follower: bob)
       expect(alice.follows?(bob)).to be_falsey
-    end 
+    end
+  end
+
+  describe "deactivate!" do
+    it "deactivates an active user" do
+      alice = Fabricate(:user)
+      alice.deactivate!
+      expect(alice).not_to be_active
```

Comment:
You should really call the expectation on `alice.reload`. That will verify that the change persists to the db.